docs(cli): add Local Proxy page (run multiple shops in parallel) (#2444)
shopware/docs: products/tools/cli/project-commands/dev-environment.md

## Further reading

> - [Local Proxy](./local-proxy.md) — run several shops at once, each on its own stable HTTPS hostname

Reviewer: ```suggestion
- [Local Proxy](./local-proxy.md) — run multiple shops in parallel, each on its own stable HTTPS hostname
```
